### ISSUE TITLE
MIN_SAFARI_VERSION=140000 is not compatible with WASM_BIGINT

### ISSUE THREAD

**Issue by ryanking13** [pyodide/pyodide#3506]
## 🐛 Bug

I am getting this warning message when building pyodide.asm.js. It seems like the two flags we are using are incompatible:

```
em++: warning: MIN_SAFARI_VERSION=140000 is not compatible with WASM_BIGINT (150000 or above required) [-Wcompatibility]
```

**Comment by rth** [comment]
Yes, the min safari version for the different flags is [set here](https://github.com/emscripten-core/emscripten/blob/529ad2c9607f4505c64caf912c5ce0c3304126d1/tools/feature_matrix.py#L28). 

We can probably bump to Safari 15.0 by now.

**Comment by hoodmane** [comment]
@sbc100 I think this message is incorrect:
https://webassembly.org/roadmap/ 
shows that the minimum safari version is for Js Bigint integration is 14.1 for desktop and 14.5 for mobile.

**Comment by hoodmane** [comment]
But I'm actually tempted to say we should enable wasm exception handling too. This forces a minimum Safari version of 15.2 and drops support for about 0.8% more users.

**Comment by ryanking13** [comment]
> But I'm actually tempted to say we should enable wasm exception handling too. This forces a minimum Safari version of 15.2 and drops support for about 0.8% more users.

+1 for forcing minimum Safari version of 15.2

**Comment by rth** [comment]
From our [docs stats](https://plausible.io/pyodide.org?period=30d&browser=Safari) Safari <15.2 was around 1% indeed last month,

![image](https://user-images.githubusercontent.com/630936/215439707-55ee29e5-9118-4471-ad6a-c5322a4f6294.png)

so dropping it sounds reasonable. 

@wlach Do you have an opinion on this? Not sure what are general best practices in this area (since we keep having this discussion every year or so when we bump the min Safari version :)  )

**Comment by ryanking13** [comment]
Not happening anymore
